jplotter/src/main/java/hageldave/jplotter/renderers/CoordSysRenderer.java

```diff
@@ -771,6 +771,15 @@ public void renderSVG(Document doc, Element parent, int w, int h) {
 		if(!isEnabled()){
 			return;
 		}
+		currentViewPort.setRect(0, 0, w, h);
+		if(isDirty || viewportwidth != w || viewportheight != h){
+			// update axes
+			axes.setDirty();
+			viewportwidth = w;
+			viewportheight = h;
+			setupAndLayout();
```

Comment:
same problem as with Legend. OpenGL context missing to close GL reseources (`setupAndLayout` misses a GLConetxtRequired annotation :/ )

Probably the same mechanism of collecting closables and closing them later should be implemented.